docker/README.md

```diff
@@ -0,0 +1,144 @@
+    hostname: mongo-db
+    container_name: db-mongo
+    ports:
+        - "27017:27017"
+    command: --bind_ip_all --smallfiles
+    volumes:
+       - mongodb:/data
+
+  orion:
+    image: fiware/orion:2.0.0
+    hostname: orion
+    container_name: fiware-orion
+    depends_on:
+        - mongodb
+    expose:
+        - "1026"
+    ports:
+        - "1026:1026"
+    command: -dbhost mongodb
+```
+
+
+
+
+## Configuration with environment variables
+
+Some of the more common variables can be configured using environment variables. The ones overriding general parameters
+in the `config.iota` set are described in the [IoTA Library Configuration manual](https://github.com/telefonicaid/iotagent-node-lib#configuration).
+
+The ones specifically relating to Ultralight bindings are described in the following table.
+
+| Environment variable      | Configuration attribute             |
+|:------------------------- |:----------------------------------- |
+| IOTA_MQTT_HOST            | mqtt.host                           |
+| IOTA_MQTT_PORT            | mqtt.port                           |
+| IOTA_MQTT_USERNAME        | mqtt.username                       |
+| IOTA_MQTT_PASSWORD        | mqtt.password                       |
+| IOTA_MQTT_QOS             | mqtt.qos                            |
+| IOTA_MQTT_RETAIN          | mqtt.retain                         |
+| IOTA_AMQP_HOST            | amqp.host                           |
+| IOTA_AMQP_PORT            | amqp.port                           |
+| IOTA_AMQP_USERNAME        | amqp.username                       |
+| IOTA_AMQP_PASSWORD        | amqp.password                       |
+| IOTA_AMQP_EXCHANGE        | amqp.exchange                       |
+| IOTA_AMQP_QUEUE           | amqp.queue                          |
+| IOTA_AMQP_DURABLE         | amqp.durable                        |
+| IOTA_AMQP_RETRIES         | amqp.retries                        |
+| IOTA_AMQP_RETRY_TIME      | amqp.retryTime                      |
+| IOTA_HTTP_HOST            | http.host (still not in use)        |
+| IOTA_HTTP_PORT            | http.port (still not in use)        |
+
+(HTTP-related environment variables will be used in the upcoming HTTP binding)
+
+More details, and further settings can be found within the IoT Agent for Ultralight [documentation](https://fiware-iotagent-ul.rtfd.io) itself.
+
+## How to build your own image
+
+The [Dockerfile](https://github.com/telefonicaid/iotagent-ul/blob/master/docker/Dockerfile) associated with this image can be used to build an image in several ways:
+
+* By default, the `Dockerfile` retrieves the **latest** version of the codebase direct from GitHub (the `build-arg` is optional):
+
+```console
+docker build -t iot-agent . --build-arg DOWNLOAD_TYPE=latest
+```
+
+* You can alter this to obtain the last **stable** release run this `Dockerfile` with the build argument `DOWNLOAD_TYPE=stable`
+
+```console
+docker build -t iot-agent . --build-arg DOWNLOAD_TYPE=stable
```

Comment:
This is fine to build the latest stable tagged release but, what about to build a particular tag, i.e. something like:

```
 docker build -t iot-agent . --build-arg DOWNLOAD_TYPE=1.6.0
```

By the way the current `if [ ${DOWNLOAD_TYPE}` works in Dockerfile I think currently it is not possible, but it would be great to modify so we can also have that possibility.

Reply:
Bash script expanded - Comments added
Fixed - 03e74ec